packages/p2p/src/components/buy-sell/buy-sell-row.jsx

```diff
@@ -194,17 +202,25 @@ const BuySellRow = ({ row: advert }) => {
                         </div>
                         <div className='buy-sell-row__rating'>
                             {!!rating_count && !!rating_average ? (
-                                <StarRating
-                                    empty_star_className='buy-sell-row__rating--star'
-                                    empty_star_icon='IcEmptyStar'
-                                    full_star_className='buy-sell-row__rating--star'
-                                    full_star_icon='IcFullStar'
-                                    initial_value={rating_average_decimal}
-                                    is_readonly
-                                    number_of_stars={5}
-                                    should_allow_hover_effect={false}
-                                    star_size={14}
-                                />
+                                <div className='buy-sell-row__rating--row'>
+                                    <Text color='less-prominent' size={isMobile() ? 'xxxs' : 'xxs'}>
+                                        {rating_average_decimal}
+                                    </Text>
+                                    <StarRating
+                                        empty_star_className='buy-sell-row__rating--star'
+                                        empty_star_icon='IcEmptyStar'
+                                        full_star_className='buy-sell-row__rating--star'
+                                        full_star_icon='IcFullStar'
+                                        initial_value={rating_average_decimal}
+                                        is_readonly
+                                        number_of_stars={5}
+                                        should_allow_hover_effect={false}
+                                        star_size={14}
+                                    />
+                                    <Text color='less-prominent' size={isMobile() ? 'xxxs' : 'xxs'}>
+                                        ({rating_count})
```

Comment:
```suggestion
                                        {rating_count}
```

Reply:
Actually i need that brackets for this:
![Screenshot 2023-04-04 at 1 41 20 PM](https://user-images.githubusercontent.com/125247833/229698315-77339af7-e1ba-49b6-9f82-23509d232556.png)

Reply:
oh I thought they are extra :laughing: